recipe/meta.yaml

```diff
@@ -18,14 +18,18 @@ build:
   skip: True  # [py<38]
   # Dropping ppc because of various build issues
   skip: True  # [(linux and ppc64le)]
-  # Only building for osx-arm64 GPU in this PR
-  skip: True  # [not (osx and arm64)]
+  # Only building for linux-amd64 GPU in this PR
+  skip: True  # [not (linux and x86_64)]
   string: gpu_cuda{{ cudatoolkit | replace('.', '') }}py{{ CONDA_PY }}h{{PKG_HASH}}_{{ PKG_BUILDNUM }}  # [(pytorch_variant == "gpu") and (linux and x86_64)]
   string: gpu_mps_py{{ CONDA_PY }}h{{PKG_HASH}}_{{ PKG_BUILDNUM }}                                      # [(pytorch_variant == "gpu") and (osx and arm64)]
   string: cpu_py{{ CONDA_PY }}h{{PKG_HASH}}_{{ PKG_BUILDNUM }}                                          # [pytorch_variant == "cpu"]
   detect_binary_files_with_prefix: False
   entry_points:
     - torchrun = torch.distributed.run:main
+  missing_dso_whitelist:
+    - "**/libcuda.so*"                # [(pytorch_variant == "gpu") and (linux and x86_64)]
+  track_features:
+    - pytorch-cpu                     # [pytorch_variant == "cpu"]
```

Comment:
Can you remind us what effect this has? I have read about the track features feature but I've never seen it in action and by reading the docs I'm not too sure how it can be used. Like how will conda know when to activate a feature?